docs/recipes/VaryNumAnimals.md

```diff
@@ -0,0 +1,47 @@
+# Varying the number of animals in multi-animal DeepLabCut
```

Comment:
The more descriptive title is great. In this case it implies an already trained model though -- this part "....than a model was trained with" -- and then the "In case of a new project" section is confusing? What if we leave "Analyzing videos with a different number of individuals" instead?